app/src/main/res/layout/activity_contributors.xml

```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:app="http://schemas.android.com/apk/res-auto"
+        xmlns:tools="http://schemas.android.com/tools"
+        tools:context=".view.activity.ContributorsActivity"
```

Comment:
`tools:context` reflects the theme of Activity set in AndroidManifest on the Android Studio preview.
I thought that it was easier to understand the preview if it had this since this application also has a theme for each Activity.
As a result, the preview has not changed in the current themes, but it may be useful when the themes are changed in the future.
[The images of this site](http://tech.pjin.jp/blog/2013/06/26/android-%E3%80%8Cxmlnstools%E3%80%8D%E3%80%8Ctoolscontext%E3%80%8D%E3%81%A3%E3%81%A6%E3%81%AA%E3%82%93%E3%81%A0%E3%82%8D%E3%81%86%EF%BC%9F%E3%80%90android-tips%E3%80%91/) helps to understand.

(Also, I personally use it when jumping from the xml file to Activity)

If you feel unnecessary, I will just delete the mistake :D